app.rb

```diff
@@ -0,0 +1,163 @@
+require_relative './classroom'
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def run
+    prompt_user
+  end
+
+  # List all books
+  def list_all_books
+    if @books.empty?
+      puts 'The book list is empty, chose 4 to create book'
+    else
+      puts @books.count <= 1 ? "\n#{@books.count} Book\n" : "\n#{@books.count} Book's \n"
+      @books.each_with_index do |book, index|
+        puts "#{index + 1} Title: '#{book.title}', Author: #{book.author}"
+      end
+    end
+  end
+
+  # List Person: Student/Teacher
+  def list_all_people
```

Comment:
- [ ] Same error on List all People. Please fix this error. 😄

![image](https://user-images.githubusercontent.com/92215878/211083224-f0c7e4a3-6fdd-4ed9-bcfe-84d7f548e927.png)